### PR TITLE
jenkinsci/jenkins is deprecated

### PR DESCRIPTION
**Pull request by ndeloof**
This change is [Reviewable](https://reviewable.io/reviews/jenkinsci/github-plugin/173)

### CODE REVIEW

File: src/test/java/com/coravy/hudson/plugins/github/GitHubRepositoryNameTest.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
 @RunWith(DataProviderRunner.class)
 public class GitHubRepositoryNameTest {
 
-    public static final String FULL_REPO_NAME = "jenkinsci/jenkins";
+    public static final String FULL_REPO_NAME = "jenkins/jenkins";
```
Comment (stephenc) [inline src/test/java/com/coravy/hudson/plugins/github/GitHubRepositoryNameTest.java]: What exactly is the point of the change here... https://github.com/jenkins/jenkins is a 404 for me

screen shot 2017-07-31 at 10 51 06

Comment (stephenc) [inline src/test/java/com/coravy/hudson/plugins/github/GitHubRepositoryNameTest.java]: @lanwen @ndeloof ^^^

**Comment by KostyaSha** [comment]
Where is the reference to description why it deprecated?

**Comment by lanwen** [comment]
This is fully cosmetical change so, why not :)

**Comment by KostyaSha** [comment]
I see you both have a lot of time.

**Comment by ndeloof** [comment]
@stephenc my bad, I made a github search on jenkinsci/jenkins for docker image, made a mistake here. @lanwen please revert.

**Comment by lanwen** [comment]
can be simply ignored

**Pull request merged**